### ISSUE TITLE
language for pdf documents?

### ISSUE THREAD

**Issue by vestmon** [michaelrsweet/htmldoc#350]
To meet accessibility compliance requirements, PDF documents must identify the language used.  I see in the notes for 1.9.5 support for `<html>` tag lang attribute. I have tried `<html lang="en">` but the generated document fails the accessibility test. Is this supported or am I doing something wrong?

**Comment by michaelrsweet** [comment]
@vestmon HTMLDOC currently does not include a Lang element in the document catalog, although that should be relatively easy to add.

**Comment by michaelrsweet** [comment]
[master c621e3c] Add Lang to document catalog (Issue #350)
